ci-operator/config/openshift/hac-dev/OWNERS

```diff
@@ -18,3 +18,6 @@ reviewers:
 - rottencandy
 - sahil143
 - vikram-raj
+- rhopp
```

Comment:
You shouldn't edit this file. This is auto generated from the source repository.

Reply:
ohh my bad let me fix it